filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1612,7 +1612,10 @@ void OpenGLDriver::makeCurrent(Handle<HwSwapChain> schDraw, Handle<HwSwapChain>
 // Updating driver objects
 // ------------------------------------------------------------------------------------------------
 
-void OpenGLDriver::setExternalIndexBuffer(Handle<HwIndexBuffer> ibh, void* externalBuffer) {
+void OpenGLDriver::setupExternalResource(intptr_t externalBuffer) {
```

Comment:
```suggestion
void OpenGLDriver::setupExternalResource(intptr_t externalResource) {
```

Reply:
Resolved in my latest commit: [fbd7516](https://github.com/shapr3d/filament/pull/46/commits/fbd7516aa11aa15ef6a7313cd86941a68660960a)